_includes/navigation.html

```diff
@@ -35,8 +35,7 @@
 							<a href="{{sublink.link}}" {% if sublink.new_window %}target="_blank" {% endif %}
 							class="subnav-item">{{sublink.name}}
 							{% if sublink.external %}
-							<img src="/images/buttons/external-link_gray.svg"
-							style="max-width:  10px; vertical-align:top;" />
+							<img src="/images/buttons/external-link_gray.svg" class="nav-sublink-logo">
```

Comment:
Always add the `alt` attributes` to the `img` tags.

Reply:
@Nikhil-Ladha I actually forgot about this open PR and your comment. I have added the alt attributes.
you can review now.